repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapper.scala

```diff
@@ -318,6 +318,10 @@ object HttpServletRequestWrapper {
     parsedParameterMap.toMap
   }
 
+  def parseQueryString(queryString: String): java.util.Map[String, Array[String]] = {
```

Comment:
This confuses me. I know I'm not on code review, but...

Why make this a method rather than a companion object function?

Also, I would probably simplify this a bit. I would add the null-safety part to the `parseParameterString` function, then have this function simply call through to `parseParameterString` then convert the result to Java.